pkg/project/testoutput/openshift/ci-operator/knative-images/discover/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # DO NOT EDIT! Generated Dockerfile for cmd/discover.
-FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.16 as builder
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.17 as builder
```

Comment:
@pierDipi maybe I'm missing something but shouldn't this be `xyz-golang-1.22-openshift-4.17` since this is for go 1.22?

Reply:
See the format https://github.com/openshift-knative/hack/pull/190/commits/2fd6f03701a1430fe54dc7de93e8f060890f2c88